mars/optimizes/tileable_graph/tests/test_column_pruning.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 1999-2020 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+import tempfile
+
+import pandas as pd
+
+import mars.dataframe as md
+from mars.config import option_context
+from mars.tests.core import TestBase
+from mars.optimizes.tileable_graph.core import get_tileable_mapping
+
+
+class Test(TestBase):
+    def testGroupByPruneReadCSV(self):
```

Comment:
We should also test 
1. two groupbyagg come after same `read_csv`.
2. `read_csv` which defines `usecols` to see if it works well.